src/ziggurat/init.clj

```diff
@@ -143,12 +143,18 @@
                            (shutdown-agents))
             "Shutdown-handler")))
 
+(declare StreamRoute)
```

Comment:
also why do we need to declare streamRoute just before it is being declared by defschema

Reply:
my repl cant load the file